docs/conf.py

```diff
@@ -0,0 +1,346 @@
+# -*- coding: utf-8 -*-
```

Comment:
Just to note that I discussed this with @funkyHat and this boilerplate is fairly standard for sphinxy stuff and very non-trivial to move to a separate repo.